### ISSUE TITLE
argon2 does not work with bun install

### ISSUE THREAD

**Issue by neo773** [oven-sh/bun#2022]
### What version of Bun is running?

0.5.6

### What platform is your computer?

Darwin 22.2.0 x86_64 i386

### What steps can reproduce the bug?

`index.js`
```javascript
const argon2 = require('argon2')
argon2.verify("hashedPassword", "password")
```
```
neo@neos-MacBook-Pro bun % node index.js
node:internal/modules/cjs/loader:998
  throw err;
  ^

Error: Cannot find module './lib/binding/napi-v3/argon2.node'
Require stack:
- /Users/neo/Desktop/temp/bun/node_modules/argon2/argon2.js
- /Users/neo/Desktop/temp/bun/index.js
```

**Comment by capaj** [comment]
@neo773 will this be fixed in next version of bun?

I got this error when I installed packages with 1.0.0 so 1.0.0 is still having this issue.
```
Error: Cannot find module '/home/capaj/work-repos/billie/lms/node_modules/argon2/lib/binding/napi-v3/argon2.node'
Require stack:
- /home/capaj/work-repos/billie/lms/node_modules/argon2/argon2.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1075:15)
    at Function.Module._resolveFilename.sharedData.moduleResolveFilenameHook.installedValue [as _resolveFilename] (/home/capaj/work-repos/billie/lms/node_modules/@cspotcode/source-map-support/source-map-support.js:811:30)
    at Function.Module._load (node:internal/modules/cjs/loader:920:27)
    at Module.require (node:internal/modules/cjs/loader:1141:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at Object.<anonymous> (/home/capaj/work-repos/billie/lms/node_modules/argon2/argon2.js:9:25)
    at Module._compile (node:internal/modules/cjs/loader:1254:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
    at Object.require.extensions.<computed> [as .js] (/home/capaj/work-repos/billie/lms/node_modules/ts-node/src/index.ts:1608:43)
    at Module.load (node:internal/modules/cjs/loader:1117:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/home/capaj/work-r
```

**Comment by neo773** [comment]
@capaj 
Bun has argon2 built in now with `Bun.password` API

**Comment by capaj** [comment]
That is nice, but I am using bun as dependency manager on my existing node.js project. My node.js project cannot be ported to bun, so I just kinda need argon binaries to install properly.
This should be reopened IMHO.

**Comment by ggobbe** [comment]
Fixed by PR #5077

The current workaround is to build dependency with npm which runs postinstall scripts:
```
npm rebuild argon2
```

**Comment by Electroid** [comment]
Duplicate of #4959
